lib/Nitrapi/Admin/Admin.php

```diff
@@ -19,7 +19,20 @@ public function getServerManager() {
         return new ServerManager($this);
     }
 
-    public function getUserToken($userId, $scopes, $serviceId = null) {
+    /**
+     * Requests a sub token for another user. This is required for calls that access the user's services.
+     *
+     * @param $userId The user to generate a token for
+     * @param array|string $scopes The scopes for the new token. Can only contain scopes of the current token.
+     * May be passed in as array or space separated string.
+     * @param null|integer $serviceId A serviceID to pin the token to. It won't be possible to access other services with this
+     * token. Passing null allows all services to be accessed.
+     * @param null|integer $expires_in The time in seconds the new token will be valid for. The life time of a sub token
+     * can never exceed the life time of the parent token. Pass in null for taking the parent token's life time.
+     *
+     * @return AccessToken
+     */
+    public function getUserToken($userId, $scopes, $serviceId = null, $expires_in = null) {
```

Comment:
I would recommend to refactor and move $userId after scopes because this field is optional and only used internal. Otherwise customer need to provide their own user id as first parameter.